### ISSUE TITLE
installing gog-galaxy from brew cask makes gog-galaxy report communication errors

### ISSUE THREAD

**Issue by nhooyr** [caskroom/homebrew-cask#42549]
#### General troubleshooting steps

- [ ] I have retried my command with `--force` and the issue is still present.
- [x] I have checked the instructions for [reporting bugs](https://github.com/caskroom/homebrew-cask#reporting-bugs) (or [making requests](https://github.com/caskroom/homebrew-cask#requests)) before opening the issue.
  - [x] None of the templates was appropriate for my issue, or I’m not sure.
- [ ] I ran `brew update-reset && brew update` and retried my command.
- [ ] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).

#### Description of issue

So I installed the gog-galaxy client with brew cask and then when launching the app, it was reporting communication errors and it would shut down.

However, installing using the .pkg file from the official site works fine.

Not sure what went wrong.

#### Output of `brew cask doctor`

```
==> Homebrew-Cask Version
Homebrew-Cask 1.4.2
caskroom/homebrew-cask (git revision ed89; last commit 2018-01-05)
==> macOS
10.13.2
==> Java
9.0.1
==> Homebrew-Cask Install Location
<NONE>
==> Homebrew-Cask Staging Location
/usr/local/Caskroom
==> Homebrew-Cask Cached Downloads
~/Library/Caches/Homebrew/Cask (11 files, 794.7MB)
==> Homebrew-Cask Taps:
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-cask (3882 casks)
==> Contents of $LOAD_PATH
/usr/local/Homebrew/Library/Homebrew/cask/lib
/usr/local/Homebrew/Library/Homebrew
/Library/Ruby/Gems/2.3.0/gems/did_you_mean-1.0.0/lib
/Library/Ruby/Site/2.3.0
/Library/Ruby/Site/2.3.0/x86_64-darwin17
/Library/Ruby/Site/2.3.0/universal-darwin17
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0/x86_64-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0/universal-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/x86_64-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/universal-darwin17
==> Environment Variables
LC_ALL="en_US.UTF-8"
PATH="/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/Homebrew/Library/Homebrew/shims/scm"
SHELL="/usr/local/bin/fish"
```

**Comment by commitay** [comment]
> However, installing using the .pkg file from the official site works fine.

We install the `pkg`.

https://github.com/caskroom/homebrew-cask/blob/dcb21868b08f47d481b70aec4d9aa520ac40e2d0/Casks/gog-galaxy.rb#L1-L12

> it was reporting communication errors and it would shut down.

Please provide screenshots or detailed information of the errors.

**Comment by nhooyr** [comment]
This is the exact error: https://www.gog.com/forum/general/connect_to_gog_galaxy_communication_service_was_lost_and_could_not_be_restored

So a few seconds after launching, a pop up comes up that says:

```
Connect to GOG Galaxy Communication Service was lost and could not be restored! The client will shutdown.
```

And then you click Ok and the client shuts down.

The one difference I noticed between installing them is that when I launched gog galaxy after installing from the website, it also asked to install some helper but I am not sure if that matters. I do not remember seeing a similar pop up when I installed from brew cask.
